sakura_core/charset/codechecker.h

```diff
@@ -388,12 +388,18 @@ char32_t ConvertToUtf32(std::wstring_view text) {
 	return 0;
 }
 
+/*!
+ * 文字列がIVSの異体字セレクタで始まっているか判定する
+ * nLenを省略した時はヌル文字の手前までが有効な文字列であると解釈する
+ */
+inline bool IsVariationSelector(const wchar_t* pStr, size_t nLen = SIZE_MAX) {
```

Comment:
```suggestion
inline bool IsVariationSelector(const wchar_t* pStr, size_t nLen) {
```
これは許容したくありません。（個人の感想です。）